linker/Linker/LinkContext.cs

```diff
@@ -245,8 +245,15 @@ public virtual void SafeReadSymbols (AssemblyDefinition assembly)
 				if (symbolReader == null)
 					return;
 
+				try {
+					assembly.MainModule.ReadSymbols (symbolReader);
+				} catch {
+					symbolReader.Dispose ();
+					throw;
```

Comment:
This is inside catch {} so just return instead of throw

Reply:
Fixed.